src/buildblock/FilePath.cxx

```diff
@@ -0,0 +1,386 @@
+#include "stir/FilePath.h"
```

Comment:
needs copyright and license info.
seems this contains code from utilities.cxx so will need copyright from there as well

Reply:
no copyright info in the .cxx file though? (I think you put it in the .h, but there it should just be your own (as it doesn't contain any code from `utilities.h` I guess)

Reply:
ok